src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: Writing Meaningful Web Content With Semantic HTML
+---
+
+import { SprkHeading } from '@sparkdesignsystem/spark-react';
+
+<span
+  id="writing-meaningful-web-content-with-semantic-html"
+  class="docs-b-Link--anchor"
+></span>
+<SprkHeading
+  element="h1"
+  additionalClasses="sprk-u-mbl sprk-u-Measure"
+  isPageTitle
+>
+  Writing Meaningful Web Content with Semantic HTML
+</SprkHeading>
+
+Typically, in a word document, writers bold or italicize text to add emphasis to the copy.
+
+When producing web content, **HTML** (Hyper Text Markup Language) **elements** define the structure, purpose and intended meaning of a page's web content.
+
+<SprkHeading
+  element="h3"
```

Comment:
Should this be an h2 and the next one be an h3 to be semantic?

Reply:
This is a good question, this page has a lot of headers; I'll review it with the UX team and get back to you.